docs/install/Fedora.md

```diff
@@ -14,7 +14,7 @@ dnf install atomic
 ##Make
 On Fedora, You need to install required build dependencies
 ```
-yum-builddep atomic
+#yum-builddep atomic
```

Comment:
sorry。I am very busy recently。i will contine test lataer。I want to run docker in special independent dir，could you teach me，how to slove this，for example，docker run --dir